src/main.cpp

```diff
@@ -1,19 +1,10 @@
-#include "wrap-hwlib.hpp"
-
-#include "camera.hpp"
-#include "greyscale_pixel.hpp"
-#include "image.hpp"
-#include "mock.hpp"
-#include "vector_2d.hpp"
-#include "ycbcr.hpp"
-int main() {
-    WDT->WDT_MR = WDT_MR_WDDIS;
-
-    hwlib::wait_ms(1000);
-    Mock camera;
-    Vector2D position(5, 10);
-    Image<GreyscalePixel> plaatje = camera.takeGreyImage();
-
-    hwlib::cout << plaatje << hwlib::endl;
-    return 0;
+#include "pi_cam.hpp"
+
+int main() {
+
+    std::cout << "Hello world!" << std::endl;
+    PiCam cam("/home/pi/Documents/Build-Env/Build-environment/modules/CAMERA/binaries/Images/", Vector2D(640, 360), 30);

```

Comment:
Absolute path, use relative path instead

Reply:
You need an absolute path. I know it's not ideal, but any user can use their own path.

Reply:
A relative path like "../binaries/Images/" should be enough, since relative paths work from the executable's directory. Which, in this case is "build/"